### ISSUE TITLE
z-index issue

### ISSUE THREAD

**Issue by TheSoulrester** [astridx/pkg_agosms#72]
in the backend (and i also think in every situation in the frontend) the z-index of the map seems to be wrong

![image](https://user-images.githubusercontent.com/7083599/112520837-2039b700-8d9c-11eb-824d-6c2d3788ccf9.png)

for frontend i already changed the z-index:

```
.leaflet-container{
    z-index: 1;
}
.leaflet-fullscreen-on {
    z-index: 999999;
}
```

**Comment by astridx** [comment]
Why do you think the z-index is wrong? What problem do you have exactly?
If you mean [this ](https://github.com/astridx/pkg_agosms/issues/70)problem, then as far as I have tested it, it is not the z-index.

**Comment by TheSoulrester** [comment]
no, the maps overlaps other visualizations as you can see in my screnshot - the map is streight above the menu "speichern, speichern & schließen, speichern und neu"

**Comment by astridx** [comment]
Oh yes, now I see that.
